meta-mender-qemu/scripts/docker/extract_fs

```diff
@@ -19,7 +19,10 @@ trap failure ERR
 trap cleanup EXIT
 
 DEV=$(losetup -f)
-IMG=$(ls /*img | head -n 1)
+IMG_PREFIX=${1}
```

Comment:
Perhaps we should have a default here? This will get used with old branches as well.

Reply:
I kind of thought that the default would be just empty `${1}` and hence an empty `IMG_PREFIX`. would you suggest something else?